fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/FDBInQueryTest.java

```diff
@@ -150,8 +150,8 @@ void testInQueryNoIndex() throws Exception {
                             .where(predicates(valuePredicate(fieldValue("num_value_2"), new Comparisons.ListComparison(Comparisons.Type.IN, ImmutableList.of(0, 2))))));
 
             assertEquals(997592219, plan.planHash(PlanHashable.PlanHashKind.LEGACY));
-            assertEquals(-951957746, plan.planHash(PlanHashable.PlanHashKind.FOR_CONTINUATION));
-            assertEquals(-739430123, plan.planHash(PlanHashable.PlanHashKind.STRUCTURAL_WITHOUT_LITERALS));
+            assertEquals(78097044, plan.planHash(PlanHashable.PlanHashKind.FOR_CONTINUATION));
+            assertEquals(290624667, plan.planHash(PlanHashable.PlanHashKind.STRUCTURAL_WITHOUT_LITERALS));
```

Comment:
All hash changes are caused by changing `isNullable` of arrays from `true` to `false`. because right now `isNullable` represents the whole array is nullable.